tests/migration_test.go

```diff
@@ -2746,7 +2746,7 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 					}
 					node.Labels[v1.HostModelCPULabel+"fake-model"] = "true"
 
-					node, err = virtClient.CoreV1().Nodes().Update(context.Background(), node, metav1.UpdateOptions{})
+					err = libnode.PatchNodeLabels(virtClient, node)
```

Comment:
Can you also fix the `AfterEach` above?

Reply:
:facepalm: that's actually why I created a function in the first place, and then I forgot...
Nice catch, fixed!